### PR TITLE
[#261] add object fit cover object

### LINKED ISSUE #261: Add object-fit-cover object
Commonly used for e.g. images/media content. Includes support for the [object-fit polyfill](https://github.com/bfred-it/object-fit-images) (JS), though that is maybe better achieved using [this post-css plugin](https://www.npmjs.com/package/postcss-object-fit-images) (does that fit our needs?).

```scss
.o-object-cover {
  display: block;
  width: 100%;
  height: 100%;
  object-fit: cover;
  font-family: 'object-fit: cover;'; // Required for the polyfill
}

```

### PR DESCRIPTION
**Pull request by planktonic**
Fixes #261 

The following changes are contained in this pull request:
- Adds the object-cover object class.
- Added to styleguide & VRT
- Polyfilled in styleguide

### Changes to objects & trumps

- [x] Styleguide documentation has been updated.
- [x] `backstop.json` file has been updated
- [x] `npm run test:build` script has been sucessfully run and new/changed images committed
- [x] `npm run stats` script run and updated `stats/css.json` commited

### Linting and testing

- [x] `npm run checks` script has been run without errors

### CODE REVIEW

File: scripts/styleguide.sh [hunk 1]
```diff
@@ -10,7 +10,9 @@ styleguide \
   --server \
   --disableEncapsulation \
   --title 'bitstyles' \
+  --extraHead '<script src="/assets/javascripts/ofi.js"></script>' \
   --extraHead '<script src="/assets/javascripts/a11y-dialog.js"></script>' \
   --extraHead '<script src="/assets/javascripts/styleguide-init.js"></script>' \
   --extraHead '<script src="/assets/javascripts/modal.js"></script>' \
   --extraHead '<script src="/assets/javascripts/expander.js"></script>' \
+  --extraHead '<script src="/assets/javascripts/object-fit-polyfill.js"></script>' \
```
Comment (planktonic) [inline scripts/styleguide.sh]: @RRorg The styleguide server doesn't have access to the `node_modules` folder, this was supposed to be checked in (like the a11y-dialog script) — I missed that it wasn’t, good catch! Updated.

**Comment by planktonic** [comment]
@RRorg I’ve rebased and fixed the issue with the polyfill script — can you take another look please?

**Comment by RRorg** [comment]
Now the polyfill script is included in the styleguide html, but the file is nowhere to be found - it seems the folder it resides in is gitignored anyway. Please double check! :)

**Comment by planktonic** [comment]
@RRorg gah, I checked in one script but not the other :/ Fixed!

**Pull request merged**